AirportGUI/src/models/AirportManager.java

```diff
@@ -61,8 +61,8 @@ public double getDistanceBetween(String str1, String str2) {
 		}
 		
 		//dummy variables
-		Airport ap1 = new Airport(str1, 0.00, 0.00, "unknown", "unknown");
-		Airport ap2 = new Airport(str2, 0.00, 0.00, "unknown", "unknown");
```

Comment:
Issue was that I created a dummy Airport object. This initializes its latitude and longitude parameters to 0.00. Needed the actual latitude and longitude parameters for this object if the object indeed exists in the map in order to get the proper distance.